platform/platform-resources-en/src/messages/SMTestsRunnerBundle.properties

```diff
@@ -18,6 +18,14 @@ sm.test.runner.ui.tests.tree.presentation.labels.no.tests.were.found.with.errors
 sm.test.runner.ui.tests.tree.presentation.labels.all.tests.passed=All Tests Passed
 sm.test.runner.ui.tests.tree.presentation.labels.all.but.ignored.passed=All Tests Passed (except ignored)
 sm.test.runner.ui.tests.tree.presentation.labels.test.noname=<no name>
+sm.test.runner.ui.tests.tree.accessible.presentation.labels.test.passed=passed
```

Comment:
`Magnitute` already contains title, why do you need to repeat it?

Reply:
This is specifically meant to capture the meaning of the left hand side icons in the test result tree. This means that the message spoken by the screen reader has to be short and to the point. It serves a very specific purpose, thus it should have its own message. That way, if other messages change for any reason, the message spoken by the screen reader does not change unintentionally. This is something that wouldn't be easily found, because so few people use a screen reader.

Reply:
It's code duplication and it would lead to the common duplication problem: when the renderer would be intentionally changed, these messages won't be changed automatically so the reader would be out-of-date.

Reply:
I think we misunderstand each other. This is not code duplication, because it serves a different purpose than the titles you see in the rest of the file, even though the words you see are mostly the same. This is meant to quickly convey the meaning of the icons on the left hand side of the test names, so the green OK or the orange icon when a test failed or was ignored. Without these, you do not know what the state of a specific test is. You only know that some test has failed, or was ignored. Good luck finding it. If the renderer changes, you would only have to change these if test outcomes change. Again, it is not the renderer that matters here, it is the test outcomes. You need the outcomes of specific tests, without that you are stuck performing every test on it's own, just to make sure that you find any errors. It is a huge time waster. I want the info from the icons, no titles, I want quick, concise information. Which is why I almost exclusively use one word.